community/getHelp.html.haml

```diff
@@ -0,0 +1,100 @@
+---
+description: Ask a question or start a discussion about JBPM
+layout: normalBase
+title: Get help
+---
+
+%h1 #{page.title}
+
+.row
+  .col-md-7
+    :asciidoc
+      == Usage question
+
+      Our friendly volunteer community regularly answers JBPM questions on StackOverflow.
```

Comment:
not sure who is actually looking after stack overflow for jbpm ... so not sure it should be there, most of what I do is over mailing lists

Reply:
yeah - it is because the link was there in the new footer - if you want to remove this link pls let me know

Reply:
I'll leave that to @krisv to decide but as far as I know no one is monitoring that

Reply:
+1, I would remove it as we really prefer people to use the mailing lists instead.